src/controller/CHIPDeviceController.cpp

```diff
@@ -1181,17 +1197,17 @@ void DeviceCommissioner::OnSessionEstablished()
     // TODO: Add code to receive OpCSR from the device, and process the signing request
     // For IP rendezvous, this is sent as part of the state machine.
 #if CONFIG_USE_CLUSTERS_FOR_IP_COMMISSIONING
```

Comment:
This flag is now gone. see https://github.com/project-chip/connectedhomeip/pull/9892

Reply:
Will keep an eye on this -- it  came back :-(